content/tools/aws-vault.md

```diff
@@ -85,20 +101,22 @@ Now we are ready to configure your AWS credentials. To add your AWS credentials
 aws-vault add example
 ```
 
-# Trouble Shooting
+# Troubleshooting
 
-Most problems are related to your environment settings.
+Most problems stem from misconfiguration.
 
-- Make sure you do not define a `[default]` profile in `~/.aws/credentials` or `[profile default]` in `~/aws/config`
-- Make sure `AWS_SDK_LOAD_CONFIG` is not set
-- Make sure `AWS_SHARED_CREDENTIALS_FILE` is not set
+- **Do not** define a `[default]` profile in `~/.aws/credentials` or `[profile default]` in `~/aws/config`
+- **Do not** set `AWS_SDK_LOAD_CONFIG`
+- **Do not** set `AWS_SHARED_CREDENTIALS_FILE`
 
-If using `--server` mode, make sure you do not have credentials exported:
+If using `--server` mode, ensure the following credentials are not exported:
```

Comment:
Also, add a caveat that they can only run one `--server` per host machine since it binds to `169.254.169.254`.   Link to this issue: https://docs.cloudposse.com/faq/aws-vault-error-failed-to-start-credential-server/